docs/topics/feed-exports.rst

```diff
@@ -602,39 +750,34 @@ Default: ``0``
 If assigned an integer number higher than ``0``, Scrapy generates multiple output files
 storing up to the specified number of items in each output file.
 
-When generating multiple output files, you must use at least one of the following
-placeholders in the feed URI to indicate how the different output file names are
-generated:
+.. setting:: FEED_EXPORT_BATCH_FILE_SIZE
 
-* ``%(batch_time)s`` - gets replaced by a timestamp when the feed is being created
-  (e.g. ``2020-03-28T14-45-08.237134``)
+FEED_EXPORT_BATCH_FILE_SIZE
+----------------------------
 
-* ``%(batch_id)d`` - gets replaced by the 1-based sequence number of the batch.
+.. versionadded:: VERSION
 
-  Use :ref:`printf-style string formatting <python:old-string-formatting>` to
-  alter the number format. For example, to make the batch ID a 5-digit
-  number by introducing leading zeroes as needed, use ``%(batch_id)05d``
-  (e.g. ``3`` becomes ``00003``, ``123`` becomes ``00123``).
+Default: ``0B``
 
-For instance, if your settings include::
+If assigned a size larger than ``0B``, Scrapy generates multiple output files based on
+the specified size as a soft limit.
 
-    FEED_EXPORT_BATCH_ITEM_COUNT = 100
+The size must be a string in the format <``SIZE``><``STORAGE-UNIT``>, where ``STORAGE-UNIT``
+must be byte unit based on powers of 2(KiB, MiB, GiB, TiB) or powers of 10(kB, MB, GB, TB).
 
-And your :command:`crawl` command line is::
+.. setting:: FEED_EXPORT_BATCH_DURATION
 
-    scrapy crawl spidername -o "dirname/%(batch_id)d-filename%(batch_time)s.json"
+FEED_EXPORT_BATCH_DURATION
+----------------------------
 
-The command line above can generate a directory tree like::
+.. versionadded:: VERSION
 
-    ->projectname
-    -->dirname
-    --->1-filename2020-03-28T14-45-08.237134.json
-    --->2-filename2020-03-28T14-45-09.148903.json
-    --->3-filename2020-03-28T14-45-10.046092.json
+Default: ``0:0:0``
 
-Where the first and second files contain exactly 100 items. The last one contains
-100 items or fewer.
+If assigned a duration longer than ``0:0:0``, Scrapy generates multiple output files based on
+the specified duration as a soft limit.
 
+Duration must be a string in the format ``hours:months:seconds``.
```

Comment:
The settings documentation should also clarify the scope, the default batch handler.

You could also remove the settings from here, mention them in the docstring of the default batch handler as equivalents to the corresponding feed option keys (indicating which one has a higher priority over the other), and move the `.. setting:: …` anchors to right above the paragraph or list item that mentions each setting.